runelite-client/src/main/java/net/runelite/client/plugins/poh/PohConfig.java

```diff
@@ -27,146 +27,186 @@
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
+import net.runelite.client.config.Units;
 
 @ConfigGroup("poh")
 public interface PohConfig extends Config
 {
-	@ConfigItem(
-		keyName = "showPortals",
-		name = "Show Portals",
-		description = "Configures whether to display teleport portals"
-	)
-	default boolean showPortals()
-	{
-		return true;
-	}
-
 	@ConfigItem(
 		keyName = "showAltar",
 		name = "Show Altar",
-		description = "Configures whether or not the altar is displayed"
+		description = "Configures whether or not the altar is displayed",
+		position = 1
 	)
 	default boolean showAltar()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showGlory",
-		name = "Show Glory mount",
-		description = "Configures whether or not the mounted glory is displayed"
+		keyName = "showDigsitePendant",
+		name = "Show Digsite Pendant",
+		description = "Configures whether or not the Digsite Pendant is displayed",
+		position = 2
 	)
-	default boolean showGlory()
+	default boolean showDigsitePendant()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showPools",
-		name = "Show Pools",
-		description = "Configures whether or not the pools are displayed"
+		keyName = "showExitPortal",
+		name = "Show Exit portal",
+		description = "Configures whether or not the exit portal is displayed",
+		position = 3
 	)
-	default boolean showPools()
+	default boolean showExitPortal()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showRepairStand",
-		name = "Show Repair stand",
-		description = "Configures whether or not the repair stand is displayed"
+		keyName = "showMagicTravel",
+		name = "Show Fairy/ Spirit Tree/ Obelisk",
+		description = "Configures whether or not the Fairy ring, Spirit tree or Obelisk is displayed",
+		position = 4
 	)
-	default boolean showRepairStand()
+	default boolean showMagicTravel()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showExitPortal",
-		name = "Show Exit portal",
-		description = "Configures whether or not the exit portal is displayed"
+		keyName = "showGlory",
+		name = "Show Glory mount",
+		description = "Configures whether or not the mounted glory is displayed",
+		position = 5
 	)
-	default boolean showExitPortal()
+	default boolean showGlory()
 	{
 		return true;
 	}
 
 	@ConfigItem(
 		keyName = "showBurner",
 		name = "Show Incense Burner timers",
-		description = "Configures whether or not unlit/lit burners are displayed"
+		description = "Configures whether or not unlit/lit burners are displayed",
+		position = 6
 	)
 	default boolean showBurner()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showSpellbook",
-		name = "Show Spellbook altar",
-		description = "Configures whether or not the Spellbook altar is displayed"
+		keyName = "notifyBurner",
+		name = "Burner Notifications",
+		description = "Configures whether or not to send a notification 5 seconds before burners enter the random timer phase",
```

Comment:
This shouldn't say that it sends a notification after 5 seconds as it seems to be configurable in the next option

Reply:
Fixed